js/cloze.js

```diff
@@ -43,7 +43,7 @@
       for (var i = 0; i < answers.length; i++) {
         // Damerau-Levenshtein comparison
         if (behaviour.acceptSpellingErrors === true) {
-          var levenshtein = H5P.TextUtilities.computeLevenshteinDistance(answered, answers[i], true);
+          var levenshtein = H5P.TextUtilities.computeLevenshteinDistance(answered, H5P.trim(answers[i]), true);
```

Comment:
@icc Pinging you as @ravimajithia is not around anymore and you merged the pull request.

The changes done in this commit use the trim() function of H5P core, not the trim() function of a string. I am pretty sure that the former was only introduced as a workaround for older versions of the Internet Explorer, so it feels very odd to still use the core version that in the code is even suggested to be declared as deprecated. Is there any particular reason that `H5P.trim(answers[i])` was used instead of `answers[i].trim()` (or respectively later on)?